### DISCUSSION TITLE
Table 4, Panel 1

### DISCUSSION THREAD

**Discussion by carolxsong** [cssi-pi-community/2022-meeting#32] [Panel 1 roundtable (Monday morning)]
Discussion board for Table 4, Panel 1

**Comment by lpottier** [comment]
### Report from the panel
Recurring theme at many conferences, we often discuss about diversity, workforce development and increasing inclusivity.
- Most institutions are doing efforts
- Students are being trained for 2-3 years and then leave for industry, salaries play an important role
- Renewal mechanism for grants are not common in NSF compared to NIH which can fund projects for 20+ years
- Lots people (mostly females) joined our community by "chance" and not via the classic pipeline 
- **Start early!** focusing efforts before college would probably be more beneficial. Focus on high-schools, middle schools workshops and activities to show what careers are possible
   - Not many fundings mechanism for that, there is REU for example from NSF.
